packagehandlers/commonpackagehandler.go

```diff
@@ -30,6 +30,8 @@ func GetCompatiblePackageHandler(vulnDetails *utils.VulnerabilityDetails, detail
 		handler = &PythonPackageHandler{pipRequirementsFile: details.PipRequirementsFile}
 	case coreutils.Maven:
 		handler = &MavenPackageHandler{depsRepo: details.DepsRepo, ServerDetails: details.ServerDetails}
+	case coreutils.Nuget, coreutils.Dotnet:
```

Comment:
This can be removed and added in the next PR, just in case your for some reason merge this one and not the second.